src-backend/api/tests/utils/decorators.py

```diff
@@ -0,0 +1,10 @@
+from api.startup import run_once
+
+
+# Use this decorator to add permissions to a new user
+# created in your setUp method for tests.
+def add_group_permissions(self):
```

Comment:
I don't like the naming for this decorator since it may one day do more than just add group permissions.